src/app/home/filter-bar/filter-bar.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-filter-bar',
+  templateUrl: './filter-bar.component.html',
+  styleUrls: ['./filter-bar.component.scss']
+})
+
+// (b/12345): Class to work with Filter-bar and it's filters
```

Comment:
and todo should look like this 
// TODO(b/12345): Class to work with Filter-bar and it's filters